drivers/source/SerialBase.cpp

```diff
@@ -32,15 +32,15 @@ SerialBase::SerialBase(PinName tx, PinName rx, int baud) :
 #endif
     _serial(),
     _baud(baud),
+    _rx_enabled(true),
+    _tx_enabled(true),
+    _tx_pin(tx),
+    _rx_pin(rx),
```

Comment:
`,` needs to be with `#if DEVICE_SERIAL_FC` or you can change the declaration order in .h

Reply:
Ah yes, missed that possibility. Good that we have CI.
Fixed it by re-ordering, force pushing a fix.

Reply:
Counter-suggestion - prevent future messing around by changing this to C++11.

Put the initialisers in the member declarations:

    #if DEVICE_SERIAL_FC
        Flow             _flow_type = Disabled;

etc

Reply:
Makes sense, definitely! Thanks!